src/setupTests.js

```diff
@@ -1,9 +1,29 @@
 import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import '@testing-library/jest-dom';
+import { setupServer } from 'msw/node';
+import { rest } from 'msw';
+
+import { OK_LOGIN_API_RESPONSE } from '~app/screens/Login/mocks';
+
+const BASE_API_URL = process.env.REACT_APP_API_BASE_URL;
+const SIGN_IN_URL = '/users/sign_in';
+const STATUS_CREATED = 201;
+
+export const server = setupServer(
+  rest.post(`${BASE_API_URL}${SIGN_IN_URL}`, (req, res, ctx) =>
```

Comment:
i think the base api url is not needed, just the signin url.
Also those url should not be here, i recommend you to create a `constants.ts` in `config/api` where you export an `ENDPOINTS` constant containing those, to reuse here and in the service